src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,92 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+import duke.task.deadline.Deadline;
+import duke.task.eventtask.EventTask;
+import duke.task.todo.ToDo;
+import duke.tasklist.TaskList;
+
+
+
+/**
+ * Storage class that deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final File file;
+    private Scanner lineReader;
+
+    /**
+     * Constructor for creating a storage variable. It will create a new tasklist.txt if it does not exist, or
+     * use existing one if it is pressent.
+     *
+     * @param file Contains the filepath to wear the tasklist.txt will be stored
+     */
+    public Storage(File file) {
+        this.file = file;
+        try {
+            file.createNewFile();
+            this.lineReader = new Scanner(file);
+
+        } catch (FileNotFoundException e) {
+            System.out.println("File is not found");
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * If a tasklist.txt file exists, this method would load it into the TaskList.
```

Comment:
Perhaps it can be started with an action word e.g. returns?
What do you think about "Loads tasks in tasklist.txt file into the TaskList if the file exists"?
I have noticed the same issue in several places too.